### PR TITLE
Add delete all breakpoints feature to debugger (javascript)

### PR DESCRIPTION
**Pull request by polaroi8d**
Now you can able to use `delete all` command, to clear the breakpoints from the whole project. 

JerryScript-DCO-1.0-Signed-off-by: Levente Orban orbanl@inf.u-szeged.hu

### CODE REVIEW

File: jerry-debugger/jerry-client-ws.html [hunk 1]
```diff
@@ -828,7 +828,23 @@
...
+        delete activeBreakpoints[i];
+        found = true;
+      }
+
+      if (!found)
+      {
+        appendLog("No active breakpoints.")
+      }
+    }
+
```
Comment (robertsipka) [inline jerry-debugger/jerry-client-ws.html]: I think this form would be better, whats your opinion?
activeBreakpoints.length != 0 ? activeBreakpoints.splice(0) : appendLog("No active breakpoints.")

Comment (polaroi8d) [inline jerry-debugger/jerry-client-ws.html]: You can't use splice() function in the activeBreakpoints object.

File: jerry-debugger/jerry-client-ws.html [hunk 2]
```diff
@@ -828,7 +828,23 @@
...
+      for (var i in activeBreakpoints)
+      {
+        delete activeBreakpoints[i];
+        found = true;
+      }
+
+      if (!found)
+      {
+        appendLog("No active breakpoints.")
+      }
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: Isn't there an easier way to check whether activeBreakpoints is empty?

File: jerry-debugger/jerry-client-ws.html (hunk 1 above)
Comment (robertsipka) [inline jerry-debugger/jerry-client-ws.html]: It's my bad, I thought that activeBreakpoints is an array.

File: jerry-debugger/jerry-client-ws.html (hunk 2 above)
Comment (robertsipka) [inline jerry-debugger/jerry-client-ws.html]: You can checked with `Object.keys(acviteBreakpoints).length`

Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: That is more complicated. Keep the bool then.

**Pull request merged**